snippets/csharp/VS_Snippets_ADO.NET/Classic WebData SqlCommand Example/CS/Project.csproj

```diff
@@ -0,0 +1,12 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
```

Comment:
Should this be a library instead? You removed the `Main` method, but there's no file with top level statements.